### PR TITLE
feat: assign asset

### PR DESCRIPTION
**Pull request by Pavithra820**
Created an API endpoint for assigning an asset to an employee.

### CODE REVIEW

File: exam_django/asset/serializers/AssignAssetSerializer.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (getwithashish) [inline exam_django/asset/serializers/AssignAssetSerializer.py:1]: Is this serializer really required?
I couldn't find it being used anywhere.

File: exam_django/asset/urls.py [hunk 2]
```diff
@@ -11,6 +11,9 @@
     AssetSearchByAssetIDView,
     MemoryView,
     EmployeeView,
+    AssignAssetView,
+
```
Comment (getwithashish) [inline exam_django/asset/urls.py:15]: Please format using black.

**Review by getwithashish — CHANGES_REQUESTED** [review]
Please make the necessary changes.

File: exam_django/asset/serializers/AssignAssetSerializer.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+
```
[the remark this answers is not in the document]
Comment (Pavithra820) [reply exam_django/asset/serializers/AssignAssetSerializer.py]: Removed all the blank spaces.

File: exam_django/asset/serializers/AssignAssetSerializer.py (hunk 1 above)
Comment (Pavithra820) [reply exam_django/asset/serializers/AssignAssetSerializer.py]: Deleted the serializer

File: exam_django/asset/urls.py (hunk 2 above)
Comment (Pavithra820) [reply exam_django/asset/urls.py]: Formatted the code using black formatter.

File: exam_django/asset/urls.py [hunk 4]
```diff
@@ -39,4 +42,5 @@
     ),
     path("memory_list/", MemoryView.as_view(), name="memory_list"),
     path("employee/", EmployeeView.as_view(), name="employeeview"),
+    path("assignasset/", AssignAssetView.as_view(),name="assignasset")
```
[the remark this answers is not in the document]
Comment (Pavithra820) [reply exam_django/asset/urls.py]: Changed the endpoint to assign_asset

File: exam_django/asset/views/AssignAssetView.py [hunk 5]
```diff
@@ -0,0 +1,158 @@
+from rest_framework import status
```
[the remark this answers is not in the document]
Comment (Pavithra820) [reply exam_django/asset/views/AssignAssetView.py:1]: Removed duplicated imports and unnecessary comments.

File: exam_django/asset/serializers/AssignAssetSerializer.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (getwithashish) [reply exam_django/asset/serializers/AssignAssetSerializer.py]: Thank you

File: exam_django/asset/serializers/AssignAssetSerializer.py (hunk 1 above)
Comment (getwithashish) [reply exam_django/asset/serializers/AssignAssetSerializer.py]: Thank you for the update.

File: exam_django/asset/urls.py (hunk 2 above)
Comment (getwithashish) [reply exam_django/asset/urls.py]: Okay.

**Review by getwithashish — APPROVED** [review]
LGTM

**Pull request merged**